lib/rucio/daemons/conveyor/common.py

```diff
@@ -440,32 +443,55 @@ def bulk_group_transfer(transfers, policy='rule', group_bulk=200, source_strateg
     return grouped_jobs
 
 
-def get_conveyor_rses(rses=None, include_rses=None, exclude_rses=None):
+def get_conveyor_rses(rses=None, include_rses=None, exclude_rses=None, vos=None):
     """
     Get a list of rses for conveyor
 
     :param rses:          List of rses (Single-VO only)
     :param include_rses:  RSEs to include
     :param exclude_rses:  RSEs to exclude
+    :param vos:           VOs on which to look for RSEs. Only used in multi-VO mode.
+                          If None, we either use all VOs if run from "def", or the current VO otherwise.
     :return:              List of working rses
     """
+    multi_vo = config_get_bool('common', 'multi_vo', raise_exception=False, default=False)
+    vo = config_get('client', 'vo', raise_exception=False, default='def')
```

Comment:
I am not sure why the client-vo setting is relevant here at all?

Reply:
The way I'd been handling RSE selection for the daemons was to limit the daemon to the VO set in the [client] section (if it was set). But to allow action across any/all VO specified if the client section doesn't have a VO set (i.e. the `super_root` at `def` is running it).
I suppose this would be unnecessary if the daemons are never run from "client" locations associated with a specific VO, or maybe this kind of check is less relevant for the conveyor than it is for the reaper(s).

Reply:
I think that should be removed. On a server/daemon machine the `[client]` section of the cfg should not have any impact on the running of the daemon, that will just lead to confusion. If needed, that should be moved to a specific section. e.g. a `vo` attribute in the `[conveyor]` section etc.

Reply:
My understanding was that server/daemon machines wouldn't have the `[client]` section, and so this part of the logic would only affect a client with daemons installed. But I suppose installing daemons on a client might not be a thing commonly done anyway? I may have got the wrong impression from the dev container, where you have access to daemons alongside the CLI / admin CLI.
I'll remove this part of the logic, and allow the daemon to run on any VO(s) passed as an argument (provided `multi_vo` is True). This happens for the replica-recoverer and reaper(s) too, so I'll make the same change in all places (and modify the tests/documentation to reflect this).